xcode/Subconscious/Shared/UIKit/BlockEditor/BlockEditorModel.swift

```diff
@@ -597,6 +775,194 @@ extension BlockEditor.Model: ModelProtocol {
         return Update(state: state)
     }
     
+    static func refreshTranscludes(
+        state: Self,
+        environment: Environment
+    ) -> Update {
+        let owner = state.ownerSphere.map({ did in Peer.did(did) })
+        
+        // Fetch only the slashlinks that are not in the cache
```

Comment:
Is it possible to hold outdated previews in the cache if indexing has occurred in the background while we're using the editor?

Reply:
Yes, in this PR, we currently make no attempt to refresh cached content after loading it. However, caching is only persistent for the lifetime of the editor panel. if you leave the editor and return to it, the content will be reloaded.